Classes/URKArchive.mm

```diff
@@ -1553,6 +1553,10 @@ - (NSString *)errorNameForErrorCode:(NSInteger)errorCode detail:(NSString * __au
             errorName = @"ERAR_UTF8_PATH_CONVERSION";
             detail = NSLocalizedStringFromTableInBundle(@"Error converting a string to UTF-8", @"UnrarKit", _resources, @"Error detail string");
             break;
+        case URKErrorCodeBadPassword:
```

Comment:
- [ ] Let's begin this case with the following line:

    URKLogError("If you're seeing this, you should be calling -[URKArchive validatePassword] before attempting to extract from a password-protected archive");

Reply:
Thank you for your comments, please see updated PR